shielddash/core/templates/shielddash/base.html

```diff
@@ -0,0 +1,17 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="google-signin-client_id" content="263509446425-83g1gv3eus8qe7ss2hgr6ec7hc6krkaf.apps.googleusercontent.com">
```

Comment:
@robhudson Can `GOOGLE_AUTH_KEY` be re-used here, or is that something different? I don't have access to see the value as far as I can tell.